src/QsCompiler/DocumentationGenerator/DocumentationWriter.cs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
```

Comment:
Since this is a new project, can nullable be enabled in the csproj instead of per file?